be/src/vec/columns/column.h

```diff
@@ -673,6 +673,10 @@ class IColumn : public COW<IColumn> {
 
     virtual void replace_column_null_data(const uint8_t* __restrict null_map) {}
 
+    virtual void replace(const Field& f, size_t self_row) {
+        LOG(FATAL) << "Method replace is not supported for " << get_name();
```

Comment:
i think let it crash or throw exception instead of implement the interface is better, since some column may not implement it, so crash is more better than implement in wrong way.